csvy/readers.py

```diff
@@ -218,7 +230,9 @@ def read_to_polars(
     import polars as pl
 
     yaml_options = yaml_options if yaml_options is not None else {}
-    header, nlines, comment = read_header(filename, marker=marker, **yaml_options)
+    header, nlines, comment = read_header(
+        filename, marker=marker, encoding=encoding, **yaml_options
+    )
```

Comment:
Same with `pl.scan_csv`

Reply:
OK so, annoyingly, pl.scan_csv only accepts "utf8" or "utf8-lossy" as encodings. "utf-8" is invalid. Which kinda messes this whole thing up 🫠

Reply:
Oh, yuck. Well I guess then for the `polars` wrapper we just have to insist that the encoding is UTF-8? We should probably document this in the docstring.